openquake/commonlib/readinput.py

```diff
@@ -1066,21 +1066,52 @@ def get_exposure(oqparam, h5=None):
     return exposure
 
 
-def read_df(fname, lon, lat, id):
+def concat_if_different(values):
+    unique_values = values.dropna().unique().astype(str)
+    # If all values are identical, return the single unique value,
+    # otherwise join with "|"
```

Comment:
`return '|'.join(unique_values)` is shorter and does exactly the same thing.

Reply:
Done